scripts/webpack/browser.js

```diff
@@ -53,46 +50,52 @@ module.exports = ({
         chunkFilename: generateName(isLegacyJS),
     },
     plugins: [
-        PROD &&
-        new AssetsManifest({
-            writeToDisk: true,
-            assets: isLegacyJS ? legacyManifestData : manifestData,
-            // Need to explicitly define output file names to avoid overwrites
-            output: isLegacyJS ? 'manifest.legacy.json' : 'manifest.json',
+        new LoadablePlugin({
+            filename: isLegacyJS ? 'loadable-stats.legacy.json' : 'loadable-stats.json',
         }),
+        PROD &&
+            new AssetsManifest({
+                writeToDisk: true,
+                assets: isLegacyJS ? legacyManifestData : manifestData,
+                // Need to explicitly define output file names to avoid overwrites
+                output: isLegacyJS ? 'manifest.legacy.json' : 'manifest.json',
+            }),
         DEV && new webpack.HotModuleReplacementPlugin(),
         DEV && new webpack.NamedModulesPlugin(),
         DEV && friendlyErrorsWebpackPlugin(),
         // https://www.freecodecamp.org/forum/t/algorithm-falsy-bouncer-help-with-how-filter-boolean-works/25089/7
         // [...].filter(Boolean) why it is used
     ].filter(Boolean),
     module: {
-        rules: [{
+        rules: [
+            {
```

Comment:
There are a lot of strange styling changes here. Do you have a different lint config?

Reply:
I feel our linting rules seem to only apply to typescript files and not javascript files, this is worth digging deeper to resolve